src/styl/co-styles.styl

```diff
@@ -340,19 +340,20 @@ $fontload-js ?= false
             tr:nth-child(2n + 1)
                 background-color alpha($co-color-primary, .2)
 
-    /* grid examples */
+    /* grid & list examples */
     .grid-demo
+    .list-demo
         padding-bottom 2em
 
         [class^=g-col-]:empty:before
+        .list-item:empty:before
```

Comment:
this can't work because `.list-item:empty:before` it will be prefixed `.co_list-item:empty:before`

Reply:
Ah ! Thank you I was wondering why it wasn't working !